README.md

```diff
@@ -30,9 +30,10 @@ jobs:
       - name: Ubiquity Bounty Bot
         uses: ubiquity/bounty-bot@c1c6c99336f34ac5e94efaed49c9f218fb7a2d76
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SUPABASE_PROJECT_URL: ${{ secrets.SUPABASE_PROJECT_URL }}
           SUPABASE_PROJECT_KEY: ${{ secrets.SUPABASE_PROJECT_KEY }}
+          FOLLOWUP_TIME: '4 days'
+          DISQUALIFY_TIME: '7 days'
```

Comment:
Shouldn't these be numbers? 
```suggestion
          FOLLOWUP_DAYS: 4
          DISQUALIFY_DAYS: 7
```

Reply:
> Shouldn't these be numbers?

We don't need to care about its unit?

Reply:
Ideally, sure. But realistically, I can't imagine that users would mandate intraday follow ups. You think they will want to have a bot hound somebody twice a day and then eliminate within the next day? lol

Reply:
I think I understand your concern about giving the user more flexibility but there is also usability to consider. Also as I mentioned earlier, I really don't see a situation where a project will request intraday follow ups. That would be micromanaging and they would probably be sending messages over Slack or whatever instead for this

Reply:
New idea, we could have two secret .env properties `FOLLOWUP_ISO` and `DISQUALIFY_ISO` if these are found, then they overwrite `FOLLOWUP_DAYS`, and `DISQUALIFY_DAYS`?

We can highlight the user friendly one to new partners, and if they ask about smaller intervals we can show them these properties? 

I was inspired to suggest this for testing purposes. Would be easier to see unassign working if it does after a few minutes.

Reply:
agree to have both `_ISO` and `_DAYS`

Reply:
based on internal conversation, we decided to go with the `ms` npm module.

Reply:
In the DMs we concluded that we will use a plain english library parser `@vercel/ms` to enhance user friendliness.